### DISCUSSION TITLE
Different description for Temurin JDKs in /usr/libexec/java_home

### DISCUSSION THREAD

**Discussion by kwin** [adoptium/adoptium-support#618] [Q&A] [answered]
When listing the current JDK via `/usr/libexec/java_home -V` on Mac OS I see the following output:

1. 19 (arm64) "Eclipse Adoptium" - "OpenJDK 19" /Library/Java/JavaVirtualMachines/temurin-19.jdk/Contents/Home
2. 18.0.2.1 (x86_64) "Eclipse Adoptium" - "OpenJDK 18.0.2.1" /Library/Java/JavaVirtualMachines/temurin-18.jdk/Contents/Home
3.  17.0.5 (arm64) "Eclipse Adoptium" - "OpenJDK 17.0.5" /Library/Java/JavaVirtualMachines/temurin-17.jdk/Contents/Home
4.  16.0.2 (x86_64) "Eclipse Temurin" - "Eclipse Temurin 16" /Library/Java/JavaVirtualMachines/temurin-16.jdk/Contents/Home
5.   11.0.16.1 (arm64) "Eclipse Adoptium" - "OpenJDK 11.0.16.1" /Library/Java/JavaVirtualMachines/temurin-11.jdk/Contents/Home

Although all versions are pretty recent they announce as either Eclipse Adoptium or Eclipse Temurin. Can this be consolidated somehow?

**Comment by karianna** [comment, marked as answer]
Eclipse Adoptium is the name of the Working Group (aka the Vendor) that provides the Eclipse Temurin Binary (the brand name of the binary). The Java 16.0.2 release had a mistake in the config
